recipes/htop/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "htop" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hishamhm/htop.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - automake
+    - gcc
+    - libtool
```

Comment:
It did not change anything on my system, so I did not add it. But let met know if I have to.